specs/regional_communities.md

```diff
@@ -1,61 +1,333 @@
+
+### Split a Community
+Instead of creating a Community from scratch, splitting an existing Community into two should be the default way of creating new communities. This is a core feature of the Community widget design.
+
+When splitting a community, the key question to ask the user is this: "Along which dimension do you want the split to occur: Geography, Language, or Constellation?"
+- If splitting along the Geography dimension then ask also if it's a sibling-split of child-split. 
+   - Example of sibling-split: [Near Sweden] splits into [Near Sweden] + [Near Norway], which becomes the sibling Community with the same parent
+   - Example of child-split: [Near Sweden] splits into [Near Sweden] + [Near Stockholm], which becomes the child Community of the preceeding community
+- If splitting along the Language dimension then 
+   - Simply ask which other language the new community should serve, which becomes a sibling Community. Example: [Near Switzerland English] splits into [Near Switzerland English] and [Near Switzerland German], both of which has the same parent [Near Eurppe English]
+- If splitting along the Constellations dimension then
+   - if there is already a constellation picked, then simply ask what constellation the new Community should serve and it becomes a sibling Community. Example: [Near Somalia DeFi] splits into [Near Somalia DeFi] + [Near Somalie Gaming]
+   - if there was no prior constellation picked, then ask which Constellations the new Community should serve and becomes a child Community. Example: [Near Uganda] splits into [Near Uganda] + [Near Uganda NFT]
+- The split has to be along at one and only one of the dimensions. The split cannot proceed unless there is an actual change proposed, one that will make the new Community different by design from the original Community from which it splits.
+
+Any member can request a split, it must not be approved by the admins. 
+- The members voting should be asked a few questions
+   - Do you approve of this split proposal?
+   - If so, would you want to become a member of the new Community? Would you voluenteer to also become and admin?
+   - If so, would you also want to stay a member of the current community?
+- It is this author's opinion that a minimum of three to five other members should approve the split and also wish to become a member of the new Community for it to take effect
+- Once sufficiently approved, then the split should immediately be executed. It should not require admin involvement or approval, because we want the split to be a natural, organic and permissionless occurence.
+
+The user requesting the split trigger an automatic vote to approve the split. If the request is approved then the original requestor will also be the first leader of the new community by default. 
+ 
+After splitting the community, then each member should be given a choice as to which of the resulting communities they wish to belong to, or if they wish to be a member of both.
```

Comment:
how this should be executed? How a user will know about it? Some sticky notification is required with option to dismiss it.

Reply:
The vote could have two questions: 1) approve the split? 2) assuming the split is approved, which resulting entity/entities would you want to belong to?